components/prism-webidl.js

```diff
@@ -0,0 +1,12 @@
+Prism.languages.webidl = Prism.languages.extend('c', {
+	'keyword': /\b(any|attribute|boolean|byte|ByteString|callback|const|deleter|dictionary|DOMString|double|enum|float|FrozenArray|getter|identifier|implements|Infinity|inherit|integer|interface|iterable|legacycaller|long|maplike|namespace|NaN|null|object|octet|optional|or|other|partial|readonly|required|sequence|serializer|setlike|setter|short|static|string|stringifier|typedef|unrestricted|unsigned|USVString|void)\b/,
+	'boolean': /\b(true|false)\b/,
+	'operator': /-|:|=|\?|\*|\//
+});
+
+Prism.languages.insertBefore('webidl', 'keyword', {
+	'class-name': {
+		pattern: /(class\s+)[a-z0-9_]+/i,
+		lookbehind: true
+	}
+});
```

Comment:
What's the point of calling `insertBefore` for webidl right after you've defined webidl? `insertBefore` is used to modify existing languages, if you're defining a language yourself you don't need it, just put `class-name` in the object literal. If you don't want to use the newer `['class-name']` syntax due to browser support, just pick a different token name without a hyphen and use `class-name` as an alias. @Golmote what do you think?
 
Also, your dependencies list C-like, but here you seem to be extending C. These are two different language definitions.